src/tree/taffy_tree/tree.rs

```diff
@@ -78,6 +82,11 @@ impl LayoutTree for Taffy {
         &self.nodes[node.into()].style
     }
 
+    #[inline(always)]
+    fn layout(&self, node: NodeId) -> &Layout {
+        &self.nodes[node.into()].layout
+    }
+
```

Comment:
I don't think this function is required. Please remove it and use `layout_mut` instead. I want to keep the `LayoutTree` trait as small as possible.

Reply:
I initially added it because I had reworked util/debug.rs, which has a non-mutable reference to the tree only, so it couldn't call `layout_mut()`. It's nice to have the debug function work for any tree, and it really doesn't need mutable access to the tree. Do you think I should modify `print_tree` to take an `&mut impl LayoutTree` anyway?

Reply:
Hmm... I see. Yeah ok, happy to accept this change then.